src/patternfly/components/ModalBox/styles.scss

```diff
@@ -2,85 +2,186 @@
 
 .pf-c-modal-box {
   // Component variables
+  $pf-c-modal-box--breakpoint-sm--Height: 400px;
```

Comment:
Can't we just hardcode these 2? if it helps to have them as sass variables, then lets have a really good comment explaining what they are and why they are here please

Reply:
Added a comment to explain why they're helpful:)